File: res/Assembly-CSharp/Android/genConvert/RaidAreaParam.py
def RaidAreaParam(json):
    this={}#RaidAreaParamjson)
    #if(json==null)
    #returnfalse
    if 'id' in json:
        this['mId'] = json['id']
    if 'order' in json:
        this['mOrder'] = json['order']
    if 'period_id' in json:
        this['mPeriodId'] = json['period_id']
    if 'boss_count' in json:
        this['mBossCount'] = json['boss_count']
    if 'area_boss_id' in json:
        this['mAreaBossId'] = json['area_boss_id']
    if 'clear_reward_id' in json:
        this['mClearRewardId'] = json['clear_reward_id']
    #returntrue
return this

File: res/Assembly-CSharp/DMM/SRPG/RaidAreaParam.cs
// Decompiled with JetBrains decompiler
// Type: SRPG.RaidAreaParam
// Assembly: Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
// MVID: BE2A90B7-A8AB-4E1F-A9DE-BBA047493101
// Assembly location: C:\r\The-Alchemist-Code-Story\res\Assembly-CSharp_japan_dmm.dll

#nullable disable
namespace SRPG
{
  public class RaidAreaParam : RaidMasterParam<JSON_RaidAreaParam>
  {
    private int mId;
    private int mOrder;
    private int mPeriodId;
    private int mBossCount;
    private int mAreaBossId;
    private int mClearRewardId;

    public int Id => this.mId;

    public int Order => this.mOrder;

    public int PeriodId => this.mPeriodId;

    public int BossCount => this.mBossCount;

    public int AreaBossId => this.mAreaBossId;

    public int ClearRewardId => this.mClearRewardId;

    public override bool Deserialize(JSON_RaidAreaParam json)
    {
      if (json == null)
        return false;
      this.mId = json.id;
      this.mOrder = json.order;
      this.mPeriodId = json.period_id;
      this.mBossCount = json.boss_count;
      this.mAreaBossId = json.area_boss_id;
      this.mClearRewardId = json.clear_reward_id;
      return true;
    }
  }
}
